From 8381098761d85ec09a838c02827df3b1dca8db6b Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@nxp.com>
Date: Wed, 8 Jul 2026 15:26:09 -0400
Subject: [PATCH 06014/15333] ARM: dts: ls1021a-moxa-uc-8410a: add led suffix
 to fix CHECK_DTBS warnings

Add led suffix to fix below CHECK_DTBS warnings:
  ls1021a-moxa-uc-8410a.dtb: leds (gpio-leds): 'beeper', ... do not match
any of the regexes: '(^led-[0-9a-f]$|led)', '^pinctrl-[0-9]+$'

Signed-off-by: Frank Li <Frank.Li@nxp.com>
---
 .../boot/dts/nxp/ls/ls1021a-moxa-uc-8410a.dts | 20 +++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm/boot/dts/nxp/ls/ls1021a-moxa-uc-8410a.dts b/arch/arm/boot/dts/nxp/ls/ls1021a-moxa-uc-8410a.dts
index d2cae8c7d7a6..8efe6b615ca8 100644
--- a/arch/arm/boot/dts/nxp/ls/ls1021a-moxa-uc-8410a.dts
+++ b/arch/arm/boot/dts/nxp/ls/ls1021a-moxa-uc-8410a.dts
@@ -40,13 +40,13 @@ reg_3p3v: regulator-3p3v {
 	leds {
 		compatible = "gpio-leds";
 
-		cel-pwr {
+		cel-pwr-led {
 			label = "UC8410A:CEL-PWR";
 			gpios = <&gpio3 27 GPIO_ACTIVE_LOW>;
 			default-state = "off";
 		};
 
-		cel-reset {
+		cel-reset-led {
 			label = "UC8410A:CEL-RESET";
 			gpios = <&gpio3 28 GPIO_ACTIVE_LOW>;
 			default-state = "off";
@@ -58,13 +58,13 @@ str-led {
 			linux,default-trigger = "mmc0";
 		};
 
-		sw-ready {
+		sw-ready-led {
 			label = "UC8410A:GREEN:SWRDY";
 			gpios = <&gpio0 18 GPIO_ACTIVE_HIGH>;
 			default-state = "on";
 		};
 
-		beeper {
+		beeper-led {
 			label = "UC8410A:BEEP";
 			gpios = <&gpio0 20 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
@@ -88,37 +88,37 @@ prog-led2 {
 			default-state = "off";
 		};
 
-		wifi-signal0 {
+		wifi-signal0-led {
 			label = "UC8410A:GREEN:CEL2";
 			gpios = <&gpio3 17 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
 
-		wifi-signal1 {
+		wifi-signal1-led {
 			label = "UC8410A:GREEN:CEL1";
 			gpios = <&gpio3 18 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
 
-		wifi-signal2 {
+		wifi-signal2-led {
 			label = "UC8410A:GREEN:CEL0";
 			gpios = <&gpio3 19 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
 
-		cpu-diag-red {
+		cpu-diag-red-led {
 			label = "UC8410A:RED:DIA";
 			gpios = <&gpio3 20 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
 
-		cpu-diag-green {
+		cpu-diag-green-led {
 			label = "UC8410A:GREEN:DIA";
 			gpios = <&gpio3 21 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
 
-		cpu-diag-yellow {
+		cpu-diag-yellow-led {
 			label = "UC8410A:YELLOW:DIA";
 			gpios = <&gpio3 22 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
-- 
2.55.0

